### PR TITLE
[New Extension] OpenAI Translator

### PR DESCRIPTION
**Pull request by douo**
## Description

[Raycast](https://www.raycast.com/) extension for translation based on ChatGPT API.

Feature:

- Translate
- Polishing
- Summarization
- [What is it?](https://github.com/bee-san/pyWhat)

## Screencast

https://user-images.githubusercontent.com/743074/224529941-84407850-f06c-4e2f-8c6f-9d835cc95408.mov

### Commands:

commands

### Translation: 
stream-text

### What is it?
what-en

### Actions

actions

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by abn3r1** [comment]
Can't wait, please launch!!!!

File: extensions/openai-translator/src/views/content.tsx [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+      </ActionPanel.Section>
+      {getLoadActionSection(record, (str) => {
+        query.updateText(str);
+      })}
+      {getModeActionSection((mode) => {
+        setMode(mode);
+      })}
+      <Action
+        title="Delete Translation"
+        icon={Icon.Trash}
```
Comment (pernielsentikaer) [inline extensions/openai-translator/src/views/content.tsx:216]:
```suggestion
        icon={Icon.Trash}
        style={Action.Style.Destructive}
```

File: extensions/openai-translator/src/views/content.tsx [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+        shortcut={{ modifiers: ["cmd"], key: "delete" }}
```
Comment (pernielsentikaer) [inline extensions/openai-translator/src/views/content.tsx:217]: Maybe use ⌃+x instead

File: extensions/openai-translator/src/views/content.tsx (hunk 1 above)
Comment (pernielsentikaer) [reply extensions/openai-translator/src/views/content.tsx:216]: Maybe even add a [confirmAlert](https://developers.raycast.com/api-reference/feedback/alert#confirmalert) if it makes sense?

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

* The extensions suffer from [Empty state flicker](https://developers.raycast.com/basics/prepare-an-extension-for-store#empty-states)  

* Where does the `Magyar` come from? I selected English as preference

Raycast-wbqwPdxE@2x

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/openai-translator/src/views/content.tsx (hunk 2 above)
Comment (douo) [reply extensions/openai-translator/src/views/content.tsx:218]: Sure. I will follow the advices.

File: extensions/openai-translator/src/views/content.tsx (hunk 1 above)
Comment (douo) [reply extensions/openai-translator/src/views/content.tsx]: Yes!

**Comment by douo** [comment]
Add a new Feature `screencaputre translation`.

**Comment by pernielsentikaer** [comment]
Which command line did you use to compile? I just compiled with `swiftc main.swift` and got another checksum `58ba5ac98cdac26e03426a28e44b687b` compared with yours `5e7350c161b592ab5713fdf6874ca5c5`

**Review by pernielsentikaer — APPROVED** [review]
Hello 👋

I gave it a quick spin, works as expected, approved 🔥

**Pull request merged**